src/js/states/mods.js

```diff
@@ -132,6 +145,32 @@ export class ModsState extends TextualGameState {
         this.app.platformWrapper.openExternalLink(THIRDPARTY_URLS.modBrowser);
     }
 
+    toggleMod(checkbox, modID, toggleIsOn) {
+        if (toggleIsOn()) {
```

Comment:
if `toggleIsOn` is immediately invoked, why not just evaluate it when calling and have its result be an argument? (Also, this function goes on to use `checkbox.classList` directly.)